client/data/disputes/hooks.ts

```diff
@@ -59,6 +59,8 @@ export const useDisputes = ( {
 	date_between: dateBetween,
 	status_is: statusIs,
 	status_is_not: statusIsNot,
+	orderby: orderBy = 'created',
```

Comment:
I'd suggest following the same approach as in `client/data/transactions/hooks.ts` and supply default as part of `const query = { ... }`

Reply:
Done!